wsd/Admin.cpp

```diff
@@ -783,6 +790,57 @@ void Admin::cleanupResourceConsumingDocs()
     _model.cleanupResourceConsumingDocs();
 }
 
+void Admin::cleanupLostKits()
+{
+    static std::map<pid_t, std::time_t> mapKitsLost;
+    std::set<pid_t> internalKitPids;
+    std::vector<int> kitPids;
+    int pid;
+    unsigned lostKitsTerminated = 0;
+    size_t gracePeriod = _defDocProcSettings.getCleanupSettings().getLostKitGracePeriod();
+
+    internalKitPids = LOOLWSD::getKitPids();
+    AdminModel::getKitPidsFromSystem(&kitPids);
+
+    for (auto itProc = kitPids.begin(); itProc != kitPids.end(); itProc ++)
```

Comment:
`for (const auto pid : kitPids)` 

No need to declare pid separately (but it's needed for the loop below, which isn't a bad thing, since sharing variables across loops without there being a dependency between them is not a good idea).

Reply:
You can see that 'pid' variable is used later in the second loop: https://github.com/CollaboraOnline/online/blob/de62c0f359ecb589d8480889092c0b6592fe56ed/wsd/Admin.cpp#L822
 I don't think it's nice to declare twice in the same function a variable with the same purpose.